selfdrive/modeld/models/dmonitoring.h

```diff
@@ -11,6 +11,8 @@ extern "C" {
 
 #define OUTPUT_SIZE 34
 
+const bool env_send_raw_dmonitor = getenv("SEND_RAW_DMONITOR") != NULL;
```

Comment:
Wait does this work in C?

Reply:
Think this works in C++ only. Functions like this get called before main. If you create objects their destructors are called when main returns.